extensions/machine-learning/package.json

```diff
@@ -155,6 +155,9 @@
     "lodash": "^4.17.21",
     "@microsoft/azdata-test": "^3.0.1"
   },
+  "resolutions": {
+    "xml2js": "^0.5.0"
```

Comment:
Did you look at whether the azure packages that use this are being updated? That would be greatly preferred over using resolutions.

Reply:
The v0.5.0 was released yesterday so I don't see all updates are available yet, I've listed packages to track better in PR description.